src/init.js

```diff
@@ -4,9 +4,17 @@ import layout from './init/layout';
 import './util/object-assign';
 
 export default function init(data) {
+    //Define new variables.
+    data.forEach(d => {
+        d.brushed = false;
+        if (d[this.config.unit_col])
+            d.measure_unit = `${d[this.config.measure_col]} (${d[this.config.unit_col]})`;
+        else d.measure_unit = d[this.config.measure_col];
+    });
+
```

Comment:
Basically going to break any measures with 2+ units in to diff panels (I think that's ok).